jerry-core/ecma/base/ecma-helpers-value.c

```diff
@@ -589,34 +586,30 @@ ecma_get_object_from_value (ecma_value_t value) /**< ecma value */
 } /* ecma_get_object_from_value */
 
 /**
- * Invert a boolean value
+ * Get pointer to error reference from ecma value
```

Comment:
@zherczeg: the docstring and function name suggests that it's getting the error from a value, but isn't it the other way around?